lectures/introduction_dynamics/linear_models.md

```diff
@@ -1266,23 +1266,23 @@ using Random
 Random.seed!(42) # For deterministic results.
 
 ϕ1, ϕ2, ϕ3, ϕ4 = 0.5, -0.2, 0, 0.5
-σ = 0.2
-
-A = [ϕ1     ϕ2    ϕ3    ϕ4
-        1.0   0.0   0.0   0.0
-        0.0   1.0   0.0   0.0
-        0.0   0.0   1.0   0.0]
-C = [σ
-        0.0
-        0.0
-        0.0]''
+sigma = 0.2
+
+A = [ϕ1 ϕ2 ϕ3 ϕ4
+     1.0 0.0 0.0 0.0
+     0.0 1.0 0.0 0.0
+     0.0 0.0 1.0 0.0]
+C = [sigma
+     0.0
+     0.0
+     0.0]''
```

Comment:
Not sure what the `''` there does.  I would try to remove it and see if it works.  If not, tell me and we can find a different replacement.